crates/aptos/homebrew/README.md

```diff
@@ -121,7 +122,7 @@ going forward, we would modify the formula slightly. See the comments below for
 
   on_linux do
     depends_on "pkg-config" => :build
-    # Might need to use "openssl@1.1", let's see https://docs.rs/openssl/latest/openssl/#automatic
+    depends_on "zip" => :build
     depends_on "openssl@3"
     depends_on "systemd"
   end
```

Comment:
Either of these OpenSSLs work, though it's platform dependent.

Only Ubuntu 22.04 really supports OpenSSL 3, the rest used 1.1 pretty much.